database/models/activityProgress.js

```diff
@@ -0,0 +1,20 @@
+module.exports = function(sequelize, Sequelize) {
+    // Define activity progress table for activities where dynamic score is applicable
+    const status = sequelize.define('activityProgress', {
+        dragAndDrop: {
+            type: Sequelize.INTEGER,
+            defaultValue: 0
+        },
+        oddOneOut: {
+            type: Sequelize.INTEGER,
+            defaultValue: 0
+        },
+        user_id: {
+            type: Sequelize.INTEGER,
+            unique: true
+        }
+    }, {
+        tableName: 'activityProgress'
```

Comment:
To maintain consistency, rename the table with underscore separation like 'activity_progress'